store/store.go

```diff
@@ -1119,6 +1128,31 @@ func findRev(needle snap.Revision, haystack []snap.Revision) bool {
 // The file is saved in temporary storage, and should be removed
 // after use to prevent the disk from running out of space.
 func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar progress.Meter, user *auth.UserState) (path string, err error) {
+
+	if useDeltas() {
+		logger.Debugf("Available deltas returned by store: %v", downloadInfo.Deltas)
+	}
+	if useDeltas() && len(downloadInfo.Deltas) == 1 {
+		workingDir, err := ioutil.TempDir("", name+"-deltas")
```

Comment:
A common pattern in go is that the err handling is right next to the err generation, so maybe switch the deltaInfo line (below) and this line?